lib/qldb.base.model.js

```diff
@@ -1,4 +1,5 @@
 const { _ } = require('lodash');
+const { toCamelCase } = require('./utils/case-convert');
```

Comment:
Question: Why do you want this function in here? What does it add?

Reply:
- this func is using for convert object-key of the passed object to camel-case
- use-case: I got different naming-standards in js-code-style (camel-case) and database naming (snake-case), ~~I plan to convert it in here~~